.gitignore

```diff
@@ -0,0 +1,7 @@
+node_modules
+.tmp
+.vscode
+dist
```

Comment:
Please also note that your email is not synced with GitHub. Please make sure you use Akvelon's email and this email is linked with your GitHub account